# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/rasterize_disabled.vert
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/rasterize_disabled.vert b/third_party/rust/spirv-cross-internal/tests/shaders/rasterize_disabled.vert
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/rasterize_disabled.vert
+++ /dev/null
@@ -1,22 +0,0 @@
-#version 450
-
-layout(location = 0) in vec4 input_buffer;
-
-layout(binding = 0, std140) uniform iu
-{
-	uvec4 m[2];
-} input_uniform;
-
-layout(binding = 1, std430) writeonly buffer ob
-{
-	uvec4 m[2];
-} output_buffer;
-
-void main()
-{
-	gl_Position = input_buffer;
-	for (int i = 0; i < 2; i++)
-	{
-		output_buffer.m[i] = input_uniform.m[i];
-	}
-}